src/cli/cluster/cluster_manager.js

```diff
@@ -86,7 +86,6 @@ module.exports = class ClusterManager {
     const fromRoot = require('../../utils/fromRoot');
 
     this.watcher = chokidar.watch([
-      'src/plugins',
```

Comment:
Maybe we should just wrap this array in a call to [`uniq`](https://lodash.com/docs#uniq)